### PR TITLE
node infobox coords link os switch

### PR DESCRIPTION
**Pull request by ghost**
Link to the "geo:"-URL on Android while linking to flopp.net otherwise

...
finally

### CODE REVIEW

**Comment by lost-tty** [comment]
I don't think this makes much sense. I believe systems other than Android may be able to parse geo-URIs and systems that don't can provide a helpful error message. One can always click on the nodename itself to see it centered on the map.

File: lib/infobox/node.js [hunk 1]
```diff
@@ -28,10 +28,18 @@ define(["moment", "numeral", "tablesort", "tablesort.numeric"],
       var latitude = d.nodeinfo.location.latitude
       var longitude = d.nodeinfo.location.longitude
       var a = document.createElement("a")
-      a.textContent = showLatitude(latitude) + " " +
+      a.innerHTML = showLatitude(latitude) + " " +
```
Comment (lost-tty) [inline lib/infobox/node.js:31]: Don't ever do this! Instead, `createTextNode`s and append to the parent.

**Comment by ghost** [comment]
The fact is that iOS doesn't throw a useful error and can't handle geo links, nor does Windows (I haven't tested that myself though). People see the link, click / tap it and just get a useless error "can't open blah blah".

**Comment by lost-tty** [comment]
Well, it doesn't break anything. In the worst case, pressing the back button will bring people back to the meshviewer.

**Comment by jplitza** [comment]
Firefox under any OS can be taught how to handle geo:-URIs, so I also use them on my desktop.

Furthermore, why on earth would you link to flopp.net? At first glance, that seems to be some kind of geocaching site, which has absolutely nothing to do with Freifunk. I would understand linking to either OSM or Google, but nothing else.

**Comment by ghost** [comment]
Flopp provides OSM and sattelite at the same time

**Comment by ghost** [comment]
Maybe @adorfer can elaborate on this

**Comment by Adorfer** [comment]
@jplitza 
Flopp provides besides different OSM styles and DOP the possibity to set markers, to make projections (distance + angle), set areals and creating shortlinks for all of this. 
e.G.: http://goo.gl/07jssy

**Comment by lost-tty** [comment]
We could change the format to something that can be copy-pasted into various forms (e.g. on flopp.net).

**Comment by Adorfer** [comment]
I forgot to mention: Additional "bonus" for flopp.net: It's on github!
https://github.com/flopp/FloppsTolleKarte

**Pull request closed without merging**